install_config/configuring_extended_ldap_attributes.adoc

```diff
@@ -0,0 +1,175 @@
+= Configuring Extended LDAP Attributes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+This topic builds upon xref:../install_config/ssd_for_ldap_failover.adoc[Setting
+up SSSD for LDAP Failover] and
+xref:../install_config/configuring_form_based_authentication.adoc[Configuring
+Form-Based Authentication] and focused on configuring extended Lightweight
+Directory Access Protocol (LDAP) attributes.
+
```

Comment:
What is an example of an extended LDAP attribute? Why would our users want to do this?